BudgetBudget/View/BudgetView.swift

```diff
@@ -38,7 +38,7 @@ struct BudgetView: View {
                         Divider()
                         ForEach(displayedMonthIDs, id: \.self) { monthID in
                             Divider()
-                            BudgetColumn(budget: budget.budgetFor(date: Date(monthID: monthID))).frame(maxWidth: .infinity)
+                            BudgetColumn(budget: budget.budgetFor(date: Date(monthID: monthID))!).frame(maxWidth: .infinity)
```

Comment:
This crashes if the monthID is before the start date. This should be prevented by the displayedMonthID not going before the start date defined in the settings.

Reply:
Fixed with latest commit